adapters/sharethrough/butler.go

```diff
@@ -50,10 +50,11 @@ type StrOpenRTBTranslator struct {
 	UserAgentParsers UserAgentParsers
 }
 
-func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openrtb.BidRequest) (*adapters.RequestData, error) {
+func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openrtb.BidRequest, domain string) (*adapters.RequestData, error) {
 	headers := http.Header{}
 	headers.Add("Content-Type", "text/plain;charset=utf-8")
 	headers.Add("Accept", "application/json")
+	headers.Add("Origin", domain)
```

Comment:
@maphe, we usually pass Site.Page as `headers.Add("Referer", domain)` and not `headers.Add("Origin", domain)`. Any specific reason why are we passing it as an `Origin` header and not a `Referer`?

Reply:
Please be advised that if your decision to fix this adapter to pull the page info from `Origin` is the correct to your particular context, we won't deny your PR for it. I'm just wondering about it because `Referer` is usually the “standard” way, but if it makes perfect sense for your adapter, we don't have a problem with this approach.

Reply:
@guscarreon the main reason is that we don't send the raw `Site.Page`, we parse it to extract the domain. `Referer` is meant to have the full URL whereas `Origin` is a more appropriate header for just a domain.